src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -3203,8 +3178,6 @@ int wm_vuldet_oval_xml_parser(OS_XML *xml, XML_NODE node, wm_vuldet_db *parsed_o
                     // Just for Ubuntu, where comment comes after test_ref
                     if (parsed_oval->vulnerabilities->state_id) {
                         if (strstr(node[i]->values[j], not_vulnerable)) {
-                            parsed_oval->vulnerabilities->ignore = 2;
-                        } else if (strstr(node[i]->values[j], needs_triage) || strstr(node[i]->values[j], deferred)) {
```

Comment:
Remove lines [3175](https://github.com/wazuh/wazuh/pull/5284/files#diff-88895483f5107f24a4e15adf36758f39R3175) and [3176](https://github.com/wazuh/wazuh/pull/5284/files#diff-88895483f5107f24a4e15adf36758f39R3176)